[2025-07-26T09:55:15.013+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T09:55:15.038+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres_test2.run_quality_checks manual__2025-07-26T09:54:56.051938+00:00 [queued]>
[2025-07-26T09:55:15.055+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres_test2.run_quality_checks manual__2025-07-26T09:54:56.051938+00:00 [queued]>
[2025-07-26T09:55:15.057+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-26T09:55:15.077+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): run_quality_checks> on 2025-07-26 09:54:56.051938+00:00
[2025-07-26T09:55:15.087+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=883) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T09:55:15.089+0000] {standard_task_runner.py:72} INFO - Started process 884 to run task
[2025-07-26T09:55:15.088+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres_test2', 'run_quality_checks', 'manual__2025-07-26T09:54:56.051938+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/my_dags.py', '--cfg-path', '/tmp/tmpmyhocpg4']
[2025-07-26T09:55:15.091+0000] {standard_task_runner.py:105} INFO - Job 41: Subtask run_quality_checks
[2025-07-26T09:55:15.181+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres_test2.run_quality_checks manual__2025-07-26T09:54:56.051938+00:00 [running]> on host 2f3d4b2b9190
[2025-07-26T09:55:15.264+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres_test2' AIRFLOW_CTX_TASK_ID='run_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T09:54:56.051938+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-26T09:54:56.051938+00:00'
[2025-07-26T09:55:15.265+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-26T09:55:15.294+0000] {my_dags.py:683} INFO - Running quality checks
[2025-07-26T09:55:15.296+0000] {my_dags.py:695} INFO - Quality check passed: staging_songs table has 71 records
[2025-07-26T09:55:15.298+0000] {my_dags.py:695} INFO - Quality check passed: staging_events table has 8056 records
[2025-07-26T09:55:15.300+0000] {my_dags.py:695} INFO - Quality check passed: songplays table has 6820 records
[2025-07-26T09:55:15.303+0000] {my_dags.py:695} INFO - Quality check passed: users table has 96 records
[2025-07-26T09:55:15.305+0000] {my_dags.py:695} INFO - Quality check passed: songs table has 71 records
[2025-07-26T09:55:15.306+0000] {my_dags.py:695} INFO - Quality check passed: artists table has 69 records
[2025-07-26T09:55:15.308+0000] {my_dags.py:695} INFO - Quality check passed: time table has 6813 records
[2025-07-26T09:55:15.309+0000] {my_dags.py:739} INFO - All quality checks passed successfully
[2025-07-26T09:55:15.310+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T09:55:15.323+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-26T09:55:15.324+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres_test2, task_id=run_quality_checks, run_id=manual__2025-07-26T09:54:56.051938+00:00, execution_date=20250726T095456, start_date=20250726T095515, end_date=20250726T095515
[2025-07-26T09:55:15.346+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T09:55:15.366+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-26T09:55:15.368+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
